realm/realm-library/src/main/java/io/realm/internal/OsResults.java

```diff
@@ -306,6 +308,10 @@ public static OsResults createFromQuery(OsSharedRealm sharedRealm, TableQuery qu
         return createFromQuery(sharedRealm, query, new DescriptorOrdering());
     }
 
+    public static OsResults createFromMap(OsSharedRealm sharedRealm, Table table, long nativePtr) {
```

Comment:
It is a bit unclear what the nativePtr is, rename to something more descriptive?